core/kazoo_documents-1.0.0/src/kz_device.erl

```diff
@@ -165,11 +165,15 @@ set_sip_settings(DeviceJObj, SipJObj) ->
 presence_id(DeviceJObj) ->
     presence_id(DeviceJObj, 'undefined').
 presence_id(DeviceJObj, Default) ->
-    wh_json:get_value(?PRESENCE_ID, DeviceJObj, Default).
+    wh_json:get_binary_value(?PRESENCE_ID, DeviceJObj, Default).
```

Comment:
presence_id(DeviceJObj, 'undefined') would return <<"undefined">> ?
how would this work with cf_endpoint merge ?